app/app_test.go

```diff
@@ -58,13 +59,15 @@ func (at *appTest) signTx(tx basecoin.Tx) basecoin.Tx {
 func (at *appTest) getTx(coins coin.Coins) basecoin.Tx {
 	tx := at.baseTx(coins)
 	tx = base.NewChainTx(at.chainID, 0, tx)
+	tx = nonce.NewTx(1, []basecoin.Actor{at.acctIn.Actor()}, tx)
```

Comment:
Okay - yeah need maj0r neatness improvements - just wanted to get something working first, but need to improve code visually before release, I've added this sequence number for more advanced testing capabilities, although I've haven't added new tests yet